CMakeLists.txt

```diff
@@ -482,6 +492,14 @@ ff_lib_enable(OPENCL OpenCL)
 ff_lib_enable(LIBCJSON libcjson)
 ff_lib_enable(FREETYPE freetype2)
 
+if(ENABLE_THREADS)
+    target_compile_definitions(libfastfetch PRIVATE FF_HAVE_THREADS)
+
+    if(CMAKE_USE_PTHREADS_INIT)
+        target_link_libraries(libfastfetch PRIVATE Threads::Threads)
+    endif()
```

Comment:
I don't think we need the if, `Threads::Threads` will be empty if threads is provided by the standard lib.